### PR TITLE
Issue 55 feat/app fortune cookie

### PR DESCRIPTION
**Pull request by Mariusz-web**

### CODE REVIEW

File: src/components/organisms/Header/Header.jsx [hunk 1]
```diff
@@ -9,22 +9,63 @@ import COOKIE from '../../../images/cookie.png';
 
 import './Header.scss';
 
+const getRandomFortune = async () => {
+    let data;
+    try {
+        const response = await fetch('/fortune/wisdom');
```
Comment (Veryyapeee) [inline src/components/organisms/Header/Header.jsx:15]: We have axios installed, use axios

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

**Comment by Mariusz-web** [comment]
I used axios instead of fetch

File: src/components/organisms/Header/Header.jsx [hunk 2]
```diff
@@ -9,22 +10,64 @@ import COOKIE from '../../../images/cookie.png';
...
+                            {text}
+                        </div>
+                    )}
+                </div>
+                <ImageComponent
+                    src={COOKIE}
+                    alt=''
+                    size='large'
+                    customClass='nav__bar__section__imgcookie'
+                    onClick={handleOnCookieClick}
```
Comment (T0ffik) [inline src/components/organisms/Header/Header.jsx:69]: Looks like clicking on cookie do nothing

File: src/components/organisms/MessageCollection/MessageCollection.jsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import PropTypes from 'prop-types';
-import { MessageComponent } from '../..';
+import { MessageComponent } from '../../molecules';
```
Comment (T0ffik) [inline src/components/organisms/MessageCollection/MessageCollection.jsx:2]: Why you changed this?

**Review by T0ffik — CHANGES_REQUESTED** [review]

File: package.json [hunk 4]
```diff
@@ -54,5 +54,6 @@
     "engines": {
         "npm": "please-use-yarn",
         "yarn": ">= 1.19.1"
-    }
+    },
+    "proxy": "http://yerkee.com/api"
```
Comment (PoProstuMieciek) [inline package.json:58]: What is this for?

File: src/components/organisms/Header/Header.jsx [hunk 5]
```diff
@@ -9,22 +10,64 @@ import COOKIE from '../../../images/cookie.png';
...
             </NavLink>
             <div className='nav__bar__section'>
-                <UserInfo onClick={handleOnImgClick} />
+                <UserInfo onClick={handleOnAvatarClick} />
                 <Navigation isOpen={isNavigationOpen} />
-                <ImageComponent src={COOKIE} alt='' size='large' onClick />
+                <div className='cookie'>
+                    {!!text && (
+                        <div className='cookie__popup'>
+                            <button
```
Comment (PoProstuMieciek) [inline src/components/organisms/Header/Header.jsx:54]: I believe that we have a custom Button component. You should use that instead

**Review by PoProstuMieciek — CHANGES_REQUESTED** [review]

**Comment by Mariusz-web** [comment]
I use Button component, how did you write @PoProstuMieciek

**Review by PoProstuMieciek — APPROVED** [review]

File: src/components/organisms/Header/Header.jsx [hunk 6]
```diff
@@ -1,30 +1,74 @@
...
 import { ReactComponent as Logo } from '../../../images/logo.svg';
 import { UserInfo } from '../../molecules/UserInfo/UserInfo';
 import { Navigation } from '../Navigation/Navigation';
 import { ImageComponent } from '../../atoms/ImageComponent/Image';
+import { Button } from '../../atoms';
 import COOKIE from '../../../images/cookie.png';
 
 import './Header.scss';
 
+const getRandomFortune = async () => {
```
Comment (Veryyapeee) [inline src/components/organisms/Header/Header.jsx:14]: That should be in separate file

File: src/components/organisms/Header/Header.jsx [hunk 7]
```diff
@@ -1,30 +1,74 @@
...
             </NavLink>
             <div className='nav__bar__section'>
-                <UserInfo onClick={handleOnImgClick} />
+                <UserInfo onClick={handleOnAvatarClick} />
                 <Navigation isOpen={isNavigationOpen} />
-                <ImageComponent src={COOKIE} alt='' size='large' onClick />
+                <div className='cookie'>
+                    {!!text && (
+                        <div className='cookie__popup'>
+                            <Button
```
Comment (Veryyapeee) [inline src/components/organisms/Header/Header.jsx:55]: Imo we should use modal component since we have it and imo it would be good fit for that

**Comment by Veryyapeee** [comment]
We have some conflicts in here, I can't actually log in, please be up to date with main

**Comment by Mariusz-web** [comment]
done

File: package.json [hunk 8]
```diff
@@ -61,5 +61,6 @@
...
```
Comment (Veryyapeee) [inline package.json:65]: Does it work on deploy? I don't think so, the real solution is to allow for cross origin access in axios request header

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

**Comment by Mariusz-web** [comment]
I can't go to deploy, but I changed

**Review by Veryyapeee — APPROVED** [review]

**Review by T0ffik — APPROVED** [review]

**Pull request merged**